src/test/java/org/opensearch/sdk/TestBaseExtension.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sdk;
+
+import org.junit.jupiter.api.Test;
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.util.List;
+
+public class TestBaseExtension extends OpenSearchTestCase {
```

Comment:
No objection to the tests that are included here, but seems like there should be more tested?  

I know a lot of `BaseExtension` is tested in HelloWorld tests, perhaps maybe just a comment here indicating that?

Reply:
Yeah I was looking at codecov results, you are right most of the code is being tested and for the ones they are not I've added them. Sure i'll add a comment.